WordPress/Classes/ViewRelated/Reader/Detail/Views/ReaderDetailNewHeaderView.swift

```diff
@@ -339,42 +342,6 @@ struct ReaderDetailNewHeaderView: View {
             .font(.footnote)
             .foregroundColor(Color(.secondaryLabel))
     }
-
```

Comment:
Ah, feels nice to see all these lines being removed! It was pretty cumbersome to recreate the two different styles for iPhone and iPad.